internal/policies/testdata/TestApplyPolicies/golden/succeed/etc/polkit-1/rules.d/00-adsys-privilege-enforcement.rules

```diff
@@ -3,5 +3,5 @@
 # Any changes will be overwritten.
 
 polkit.addAdminRule(function(action, subject){
-	return ["unix-group:sudo","unix-group:admin","unix-user:alice@domain","unix-user:bob@domain2","unix-group:mygroup@domain","unix-user:cosmic carole@domain"];
+	return ["unix-user:alice@domain","unix-user:bob@domain2","unix-group:mygroup@domain","unix-user:cosmic carole@domain"];
```

Comment:
We didn't test parsing local admins in this general ApplyPolicy test (see that it is the general policy manager test rather than the specific one). The fluke here was having the local admins (since we weren't overriding the file, we were parsing the ones in the local machine and this isn't possible in Launchpad).